<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ironman</class>
 <widget class="QMainWindow" name="Ironman">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ironman Logs</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableView" name="tableView">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>&amp;Add...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editButton">
          <property name="text">
           <string>&amp;Edit...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="text">
           <string>&amp;Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>743</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Sport">
    <property name="title">
     <string>&amp;Sport</string>
    </property>
    <addaction name="action_Bike"/>
    <addaction name="action_run"/>
    <addaction name="action_Swim"/>
   </widget>
   <widget class="QMenu" name="menu_Reports">
    <property name="title">
     <string>&amp;Reports</string>
    </property>
    <addaction name="action_Summary"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Sport"/>
   <addaction name="menu_Reports"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Bike">
   <property name="text">
    <string>&amp;Bike</string>
   </property>
  </action>
  <action name="action_run">
   <property name="text">
    <string>&amp;Run</string>
   </property>
  </action>
  <action name="action_Swim">
   <property name="text">
    <string>&amp;Swim</string>
   </property>
  </action>
  <action name="action_Summary">
   <property name="text">
    <string>&amp;Summary</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>Ironman</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
